client/src/env.ts

```diff
@@ -8,21 +8,25 @@ export const DISABLE_AUTH = Boolean(
   JSON.parse(process.env.REACT_APP_DISABLE_AUTH || "false")
 );
 
```

Comment:
Follow-up thought: Should we emit a `console.warn()` if `process.env.REACT_APP_CRUD_MODE` is set, which points to WRITER_MODE/DEV_MODE?

Reply:
Perhaps we deprecate the property, issuing this console message, but still let it set REACT_APP_WRITER_MODE for now? Then schedule removing both in the future.